molten-http-client/src/main/java/com/hotels/molten/http/client/OkHttpCallFactoryFactory.java

```diff
@@ -141,4 +146,13 @@ private void addEventTracing(OkHttpClient.Builder clientBuilder, RetrofitService
             clientBuilder.eventListenerFactory(DelegatingEventListener.delegatedFactory(eventListenerFactories)::apply);
         }
     }
+
+    private List<okhttp3.Protocol> getProtocol(List<Protocol> protocol) {
+        return protocol
+            .stream()
```

Comment:
[format] Please remove unnecessary new line, same for okhttp factory
```suggestion
        return protocol.stream()
```